### PR TITLE
Add Stack Pack guidance to Lighthouse docs

### LINKED ISSUE #3674: Incorporating Stack Packs into web.dev
cc: @housseindjirdeh @paulirish @egsweeny 

### What page(s) need to be updated?

All Lighthouse [Opportunities](https://web.dev/lighthouse-performance/#opportunities) and [Diagnostics](https://web.dev/lighthouse-performance/#diagnostics) pages in web.dev that have applicable stack pack recommendations.

### Why is this update needed?

Stack packs are Google approved, framework specific suggestions that appear in Lighthouse audits within the Opportunities, Diagnostics, and Passed Audits tab. However, they aren't mentioned in the relevant web.dev articles. This would make the web.dev resources more complete, provide a space for additional elaboration on framework specific recommendations, and surface stack pack recommendations/best practices in more places for users to discover.

### What's the deadline?

None, just want to start a discussion :)

### PR DESCRIPTION
**Pull request by kaycebasques**
Continuation of #4093 

fixes #3674

### CODE REVIEW

File: src/site/content/en/lighthouse-performance/efficient-animated-content/index.md [hunk 1]
```diff
@@ -64,6 +65,19 @@ Luckily, you can recreate these behaviors using the `<video>` element.
 </video>  
 ```
 
+## Use a service that converts GIFs to HTML5 videos
+
+Many [image CDNs](/image-cdns/) support GIF to HTML5 video conversion. You upload a
+GIF to the image CDN, and the image CDN returns an HTML5 video.
+
```
Comment (kaycebasques) [inline src/site/content/en/lighthouse-performance/efficient-animated-content/index.md:72]: @paulirish in #4093 you had this listed as stack-specific guidance for Drupal, Joomla, and WordPress. The fact that the text was the same in each section suggests that it's not stack-specific guidance (by definition) so I moved it up into its own section.

https://github.com/GoogleChrome/web.dev/pull/4093/files#diff-7e25d666eee78bb880715d980b8cee454af0c39a7505421d3bc77e6828cc3614R75

File: src/site/content/en/lighthouse-performance/unminified-css/index.md [hunk 2]
```diff
@@ -75,6 +75,34 @@ This is usually accomplished with a build tool like Gulp or Webpack.
 
 Learn how to minify your CSS code in [Minify CSS](/minify-css).
 
+## Stack-specific guidance
```
Comment (kaycebasques) [inline src/site/content/en/lighthouse-performance/unminified-css/index.md:78]: @paulirish you originally had an AMP section here. But the link doesn't seem relevant to the topic of minification. https://github.com/GoogleChrome/web.dev/pull/4093/files#diff-ed9d24644cc976916d01f6513596a1465d8b32204949f1d5e01bef6fe536f2f8R82

**Comment by kaycebasques** [comment]
@paulirish do we need to manually update the Lighthouse UI to link to these sections?

**Comment by paulirish** [comment]
> @paulirish do we need to manually update the Lighthouse UI to link to these sections?

nope. the generic LH link will continue pointing to the page itself. 
(And this text that's now integrated here is already in the Lightouse report UI)

**Review by paulirish — APPROVED** [review]
your copyedits look good. 👍 

once this lands i'll port your edits back to the stackpack repo so they end up in the lighthouse report.

**Comment by kaycebasques** [comment]
@paulirish FYI I just merged

I didn't know this text was in Lighthouse UI. I worry we'll get out-of-sync but we can cross that bridge when we get there.
